### PR TITLE
walkthrough: Add command hint to remove token from Bash history

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: walkthrough/index.html [hunk 1]
```diff
@@ -39,7 +39,10 @@
 
 After hitting the `Enter` key, you should see a notification from the CLI that a configuration file has been written.
 
-### Important: The token above should be treated like a password and not be shared with anyone!
+### Important: The token above should be treated like a password and not be shared with anyone.
+   For this reason, you can delete it from your shell history, if you are using Bash, by running:
```
Comment (SleeplessByte) [inline walkthrough/index.html:43]:
```suggestion

   For this reason, you can delete it from your shell history, if you are using Bash, by running:
```

File: walkthrough/index.html [hunk 2]
```diff
@@ -39,7 +39,10 @@
...
+   `for X in $(history | grep "exercism configure --token=" | grep -v grep | cut -d ' ' -f1) do; history -d $X; done`
```
Comment (SleeplessByte) [inline walkthrough/index.html:44]:
````suggestion
   ```bash
   for X in $(history | grep "exercism configure --token=" | grep -v grep | cut -d ' ' -f1) do; history -d $X; done
   ```
````

**Comment by SleeplessByte** [comment]
cc @exercism/product

**Comment by iHiD** [comment]
@eksperimental Thanks for this. This needs fixing upstream in https://github.com/exercism/interactive-cli-walkthrough - this file is auto-generated from that repo. Could you add the fix there pls and tag @kntsoriano in it and we'll get it merged this week 🙂

**Pull request closed without merging**